src/core/client.ts

```diff
@@ -76,6 +76,7 @@ interface Attachment {
 
 export interface ClientOptions {
   key?: string;
+  meta?: Map<string, string>;
```

Comment:
Do JavaScript developers use `map` a lot?
I think it would be more convenient if the type of meta was declared as interface Meta {[key: string]: string} because the user could use the Object literal.
However, this may be my prejudice that arises because I haven't used Map often while developing. Let me know your opinion :)

Reply:
Yes, I agree with your opinion. The API seems to be easier if users use object literals.